services/batch/service/src/main/java/org/collectionspace/services/batch/nuxeo/UpdateObjectLocationBatchJob.java

```diff
@@ -76,15 +75,17 @@ public UpdateObjectLocationBatchJob() {
     }
 
     @Override
-    public void run() {
-        run(null);
+    public void run(BatchCommon batchCommon) {
+        String errMsg = String.format("%s class does not support run(BatchCommon batchCommon) method.",
+                                      getClass().getName());
+        throw new UnsupportedOperationException(errMsg);
     }
 
     /**
      * The main work logic of the batch job. Will be called after setContext.
      */
     @Override
-    public void run(BatchCommon batchCommon) {
+    public void run() {
```

Comment:
According to [DRYD-878](https://collectionspace.atlassian.net/browse/DRYD-878),  `run()` is deprecated, and `run(BatchCommon)` is preferred. So this is a step backwards, and this change should be reverted.

Reply:
Sure. Perhaps this was to support existing/older batch jobs? I don't recall. If so, we could at least add @deprecated annotation?